graviti/manager/branch.py

```diff
@@ -127,6 +133,14 @@ def get(self, name: str) -> Branch:
             self._dataset.name,
             branch=name,
         )
+
+        if (
+            hasattr(self._dataset, "HEAD")
```

Comment:
In what condition, the `self._dataset` does not have `"HEAD"`

Reply:
initialize the dataset in `ws.datasets.get()` and `ws.datasets.list()`

Reply:
Can we make OpenAPI list/get datasets return the default branch details

Reply:
The backend also needs to call "get branch" again. So I thought it was not necessary to change.